packages/contracts/test/deploy/updateTo1_3_0.ts

```diff
@@ -1,49 +0,0 @@
-import {
-  ForkOsxVersion,
-  initForkForOsxVersion,
-  initializeDeploymentFixture,
-} from '../test-utils/fixture';
-import {activeContractsList as v1_0_0_activeContracts} from '@aragon/osx-ethers-v1.2.0';
-import {expect} from 'chai';
-import {deployments} from 'hardhat';
-
-const enableTest = process.env.TEST_UPDATE_DEPLOY_SCRIPT !== undefined;
-const network = 'mainnet';
-
-if (enableTest) {
-  describe('update/to_v1.3.0', function () {
-    before(async () => {
-      const previousOsxVersion: ForkOsxVersion = {
-        version: 'v1.0.1',
-        activeContracts: v1_0_0_activeContracts,
-        forkBlockNumber: 16722881,
-      };
-
-      await initForkForOsxVersion(network, previousOsxVersion);
-
-      const updateDeployTags = ['v1.3.0'];
-      await initializeDeploymentFixture(updateDeployTags);
-    });
-
-    it('deploys new contracts with new addresses', async function () {
-      const changedContracts = [
-        'DAOFactory',
```

Comment:
Why is this deleted with the DAOFactory in it

Reply:
We won't do the update to 1.3.0 anymore and the "tests" in here aren't really testing anything.
We just check that some contracts were deployed and 4/5 we don't deploy anymore.
We should think about better deployment tests in the future.

Reply:
fine then leave it, I just wanted to check